modules/tensor_mechanics/tests/mohr_coulomb/tests

```diff
@@ -126,6 +133,13 @@
     rel_err = 1.0E-5
     abs_zero = 1.0E-3
   [../]
+  [./uni_axial1_small_strain]
+    type = 'CSVDiff'
+    input = 'uni_axial1_small_strain.i'
+    csvdiff = 'uni_axial1.csv'
+    rel_err = 1.0E-5
+    abs_zero = 1.0E-3
```

Comment:
We have somehow lost the "Initial Residual" in our output since we've overhauled everything. I believe we call this our "reference residual" and it is the value that we compare to for convergence. This reference residual is calculated before applying present boundary conditions. If your problem doesn't have any PresetBCs then this value should be the same as the first nonlinear residual. If you do however it can be several magnitudes of order "smaller" than the first nonlinear residual.

@friedmud - Is there a reason we don't use the residual with the Presets applied for our convergence check? In problems like this, it makes tuning the nonlinear solver next to impossible. @aeslaughter before we go adding the reference residual back into the output, I argue this wouldn't be necessary if it matched the first nonlinear residual and would therefore just be duplicate information printed to the screen. Am I missing something?